packages/cli/src/commands/org/resources/preview.ts

```diff
@@ -39,6 +39,7 @@ export default class Preview extends Command {
   public static flags = {
     ...wait(),
     ...sync(),
+    // TODO: CDX-935 add flag to automatically transfer vault entries when transferable
```

Comment:
For push, sure, it'll be the `-y` flag.
Outta rethink abt it for preview.
I'm not sure it's really a good idea to have flags to skip stuff for the preview yet.

Too much options is as bad as not enough so let's find the good middle

Reply:
We could have one flag (e.g. `--yes`) to automatically approve all questions (similar to [Pulumi](https://www.pulumi.com/docs/reference/cli/pulumi_up/#options)).

Reply:
I'm still unsure for the preview due to its nature that is fundamentally oriented towards a human, and thus where interactions are not bad if well spaced.
The `--yes` flag is really there for our cold metal overlord
![image](https://i.gifer.com/origin/e4/e4ea520e58f46033bf97d92c3e7e7609.gif)